### PR TITLE
Ui threshold scheduling

### PR DESCRIPTION
**Pull request by 50vincents**

### CODE REVIEW

**Comment by jiyingqi** [comment]
I tested the UI and it looks a lot cleaner, although I think we can increase the font size. There is a bit too much free space everywhere and I think we could make the app more readable.

File: QuietPlace/App.js [hunk 1]
```diff
@@ -43,4 +42,16 @@ const styles = StyleSheet.create({
       alignItems: 'center',
       justifyContent: 'center',
    },
+   homelogo: {
+      fontWeight: 'bold', 
+      fontStyle: 'normal',
+     // fontFamily: 'Roboto',
+      fontSize: 16,
```
Comment (jiyingqi) [inline QuietPlace/App.js]: This could be larger!

File: QuietPlace/App.js [hunk 2]
```diff
@@ -43,4 +42,16 @@ const styles = StyleSheet.create({
       alignItems: 'center',
       justifyContent: 'center',
    },
+   homelogo: {
```
Comment (jiyingqi) [inline QuietPlace/App.js]: Home logo I think could be placed further up, there is too much space from the top of my phone (S10+) to the logo.

**Comment by jleckron** [comment]
I agree with Jiying that there could be less free space. But I don't think the font should be too much larger. Just enough to fill the width of the screen.

File: QuietPlace/microphoneListener.js [hunk 3]
```diff
@@ -64,34 +81,69 @@ export default class MicrophoneListener extends Component {
...
+                step = {1}
+                minimumValue = {-160}
+                maximumValue = {0}
+                thumbTintColor = '#C4C4C4'
+                minimumTrackTintColor = 'black'
+                maximumTrackTintColor = 'grey'
+              />
+          
+              <TouchableOpacity style={styles.button}>
+                <Text style={styles.buttonText}>Settings</Text>
```
Comment (jleckron) [inline QuietPlace/microphoneListener.js]: I think we should look into a way of making UI elements fill the width of whatever screen it is being shown on.

**Comment by renaidisco** [comment]
The UI positioning looks fine. The button color for the settings should be lighter to have more contrast with the background.

As for the empty space, I don't think it's a concern right now since we will be adding current sound input visuals on there.

Same as Akihirio: the button is not functioning yet, right? Just want to make sure I got it down correctly.

File: QuietPlace/microphoneListener.js [hunk 4]
```diff
@@ -35,20 +35,37 @@ export default class MicrophoneListener extends Component {
...
+            fiveSoundFrames.push(data.value);
+            count++;
+            const avg = fiveSoundFrames.reduce((p, c, _, a) => p + c/a.length, 0);
+            console.log(avg);
+            console.log(notificationPause);
+            if(notificationPause < 20){
+              notificationPause++;
+            }
+            if(count == 5 && avg >= this.state.value && notificationPause == 20){
+              fiveSoundFrames = new Array(5);
```
Comment (renaidisco) [inline QuietPlace/microphoneListener.js]: I think the number of ticks in the average sound level detection should be decreased from 5 to 3.
3 feels more appropriate for spikes in volume without being as overbearing as 1.
5 feels a bit too forgiving.

**Comment by jiyingqi** [comment]
I like these changes! I think this branch is ready to be merged

**Pull request merged**